rundeckapp/grails-app/services/rundeck/services/FrameworkService.groovy

```diff
@@ -130,6 +147,43 @@ class FrameworkService implements ApplicationContextAware, ClusterInfoService {
         System.getProperty(SYS_PROP_SERVER_ID)
     }
 
+    /**
+     * @return the log global config from framework.properties file
+     */
+    boolean getProjectLogGlobalConfig() {
+        String res = rundeckFramework
+                .getPropertyRetriever()
+                .getProperty(PROJECT_LOG_GLOBAL_CONFIG) ?: Boolean.toString(false)
+        return res.toBoolean()
+    }
+
+    /**
+     * @return the log output limit size from framework.properties file
+     */
+    String getProjectLogOutputLimit() {
+        return rundeckFramework
+                .getPropertyRetriever()
+                .getProperty(PROJECT_LOG_OUTPUT_LIMIT)
+    }
+
+    /**
+     * @return the log limit action from framework.properties file
+     */
+    String getProjectLogLimitAction() {
+        return rundeckFramework
+                .getPropertyRetriever()
+                .getProperty(PROJECT_LOG_LIMIT_ACTION)
+    }
+
+    /**
+     * @return the log size warning from framework.properties file
+     */
+    String getProjectLogSizeWarning() {
+        return rundeckFramework
+                .getPropertyRetriever()
+                .getProperty(PROJECT_LOG_SIZE_WARNING)
+    }
+
```

Comment:
avoid using framework properties for global config.  it should use configurationService to retrieve config values

Reply:
Done